plan/test_plan.py

```diff
@@ -611,11 +611,12 @@ def test_available_plans_for_users_trial_plan_ongoing_trial_more_than_10_users(
         expected_result = []
         expected_result.append(BASIC_PLAN)
         expected_result += PR_AUTHOR_PAID_USER_PLAN_REPRESENTATIONS.values()
+        expected_result += TEAM_PLAN_REPRESENTATIONS.values()
```

Comment:
Oh this shouldn't happen actually, if someone is trialing, we want to only offer the team plan if there are LTE 10 plans. While they're trialling, we only show them the option for team bw 0 and 10 users